source/_integrations/nextdns.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: NextDNS
+description: Instructions on how to integrate NextDNS within Home Assistant.
+ha_category:
+  - Network
+ha_release: 2022.8.0
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@bieniu'
+ha_domain: nextdns
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+NextDNS is a DNS service that protects from all kinds of security threats, blocks ADS and trackers on websites and in apps and provides a safe and supervised Internet for kids — on all devices and on all networks. The NextDNS integration allows you to monitor your NextDNS statistics.
+
+## Setup
+
+To obtain API key go to the [NextDNS](https://nextdns.io/) site >> Account section.
```

Comment:
When I visited  https://nextdns.io/ it took me awhile to see an account section.  I then went to  https://my.nextdns.io/ but then still took me awhile to find that it was underneath the email address in the top right. Maybe most users figure this out faster than me :) but would it make sense to link to https://my.nextdns.io/account? I realize sites can change, etc,  but was just thinking it would help.

Reply:
Good idea, I'll add a link.